core/src/main/java/org/infinispan/distribution/ch/ConsistentHash.java

```diff
@@ -41,6 +41,11 @@
  */
 public interface ConsistentHash {
 
+   public static final String STATE_CONSISTENT_HASH = "consistentHash";
+   public static final String STATE_HASH_FUNCTION = "hashFunction";
+   public static final String STATE_MEMBER = "member.%d";
+   public static final String STATE_MEMBERS = "members";
```

Comment:
Should these really be in the interface? Must implementers use these particular keys?